File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // get height information from customer.
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while ((height < 1) || (height > 8));

    // first level of loop for rows
    for (int r = 0; r < height; r++)
    {
        // 1/3 in second level of loop for blanks on the left side
        for (int c = (height - r - 1); c > 0; c--)
        {
            printf(" ");
        }

        // 2/3 in second level of loop for # on the left side
        for (int c = 0; c <= r; c++)
        {
            printf("#");
        }

        printf("  ");

        // 3/3 in second level of loop for # on the right side
        for (int c = 0; c <= r; c++)
        {
            printf("#");
        }

        printf("\n");
    }

}

File: sentimental-mario-more/mario.py
# TODO: print Mario bricks using python
from cs50 import get_int

# Get input: int between 1-8
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

# Loop to print # with the given number
for i in range(1, height+1):
    # To print blanks for each first half
    for j in range(1, height-i+1):
        print(" ", end="")

    # To print # for each first half
    for j in range(1, i+1):
        print("#", end="")

    print("  ", end="")

    # To print # for each second half
    for j in range(1, i+1):
        print("#", end="")
    print()
